cloudify/tests/test_context.py

```diff
@@ -668,7 +668,6 @@ def get_template(ctx, testing, **_):
                 rendering_tests_demo_conf,
                 template_variables={'ctx': ctx})
         except exceptions.NonRecoverableError:
-            print 'caught expected exception'
```

Comment:
Shouldn't we leave the print in, just as a function?

Reply:
(ditto for other prints)

Reply:
no, those prints do nothing, they're in the tests and there's mostly no way to recover the log that those prints go into anyway; if someone needs logs there (which they tbh shouldn't) they are welcome to add a ctx.logger call instead

Reply:
Sounds fair, marking resolved.